examples/static-file-server/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+use axum::http::StatusCode as SC;
+use axum::{body::Body, http::Request};
+use http_body_util::BodyExt;
+use tower::ServiceExt;
+
+const INDEX_HTML: &str = include_str!("../assets/index.html");
+const SCRIPT_JS: &str = include_str!("../assets/script.js");
+
+const JS: &str = "text/javascript";
+const HTML: &str = "text/html";
+
+async fn get_page(app: Router, path: &str) -> (StatusCode, String, String) {
+    let response = app
+        .oneshot(Request::builder().uri(path).body(Body::empty()).unwrap())
+        .await
+        .unwrap();
+
+    let status = response.status();
+    let content_type = match response.headers().get("content-type") {
+        Some(content_type) => content_type.to_str().unwrap().to_owned(),
+        None => String::new(),
```

Comment:
Empty header value is not the same thing as a missing header so I'd generally not recommend anyone to do this. returning `Option` would be more correct.

You even later in this file check for `""` which probably is trying to check for the missing header.